Modelica/Mechanics/Translational.mo

```diff
@@ -2858,9 +2858,9 @@ where the different effects are visualized:
       extends Modelica.Thermal.HeatTransfer.Interfaces.PartialElementaryConditionalHeatPortWithoutT;
 
       parameter Real f_pos[:, 2]=[0, 1]
-        "[v, f] Positive sliding friction characteristic (v>=0)";
+        "Positive sliding friction characteristic [N] as function of v (col. 1, v>=0)";
```

Comment:
You mention the unit `[N]` here, but use `unit=N.m` for the rotational case. Also, it says col. 1, which you removed in the rotational case after my previous review comment. Would be good to also mention the unit of v then.

I believe it is not used consistently in MSL, but I'd propose to have it as `"Positive sliding friction characteristic in [N] as piecewise linear function of v in [m/s] (v>=0)"` and similar for all other cases.

Reply:
Fixed, but without "piecewise linear" as this is only a tabled data input without any relevance for the interpolation. In the documentation of the classes, this is indeed given: "... only linear interpolation in the table is supported".